### DISCUSSION TITLE
玩陸板的該怎麼轉繁體

### DISCUSSION THREAD

**Discussion by Tiurf2g** [GpointChen/FFXIVChnTextPatch-GP#102] [General]
試過將簡體轉繁體，以為成功了，結果在DX11上出了問題，只能再重新下載回簡體XD
有沒有方法可行呢?
謝謝

**Comment by GpointChen** [comment]
我沒有玩陸版，但我知道陸版檔案和國際服有所差異，不確定這個程式能不能適用。

你是漢化時是怎麼設置的？確切遇到什麼問題？
